src/image_formats/webp.nim

```diff
@@ -0,0 +1,79 @@
+import std/[streams,bitops,strutils,endians]
+
+
+proc get_size_from_vp8_header_block( stream: var FileStream ): (int,int)
+proc get_size_from_vp8l_header_block( stream: var FileStream ): (int,int)
+proc get_size_from_vp8_header_block( stream: var FileStream ): (int,int)
+
+proc get_size*( path:string ): (int,int) =
+  var stream = new_file_stream( path, fm_read )
+  defer: stream.close()
+
+  stream.set_position(12)
```

Comment:
Can we do something with these magic numbers? There are a bunch of them in this pull request.

Reply:
Open to suggestions 🙂